src/components/dynamic-label/dynamic-label.scss

```diff
@@ -25,8 +23,9 @@ limel-icon {
 }
 
 span {
-    @include mixins.truncate-text;
     flex-grow: 1;
     font-size: 0.8125rem; // `13px`, Like Checkbox & Switch
+    letter-spacing: 0.02em;
     color: var(--mdc-theme-on-surface);
```

Comment:
This row: `color: var(--mdc-theme-on-surface);` makes the label another color than the checkbox label.
Not sure it should be removed or not, but it is slightly different I think